sentences/nb/_common.yaml

```diff
@@ -66,7 +66,6 @@ expansion_rules:
   garasjedør: "garasj[edør|eport][(en|er|ene)]"
   gardin: "[rulle]gardin[(et|er|ene)]|persienn[et|er|ene]|forheng[et|er|ene]"
   persienne: "persienne[(n|r|ene)]"
-  rullegardin: "rullegardin[(et|er|ene)]"
```

Comment:
I can see we don't need this line, but shouldn't the line before (`persienne`) also be removed? It's included in `gardin`.

Reply:
Thank you. I have removed persienne. We might want to reintroduce rullegardin/persienne in the event we want different responses for device_class curtain vs blinds.